s2i-dotnet/README.md

```diff
@@ -0,0 +1,94 @@
+# Source-to-Image
+
+[Source-to-Image (S2I)](https://github.com/openshift/source-to-image)
+is a toolkit and workflow for building reproducible container images
+from source code. S2I produces images by injecting source code into a
+base S2I container image and letting the container prepare that source
+code for execution. The base S2I container images contains the
+language runtime and build tools needed for building and running the
+source code.
+
+## Install the Dotnet Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/openshift/pipelines-catalog/master/s2i-dotnet/s2i-dotnet-task.yaml
+```
+
+## Inputs
+
+### Parameters
+
+* **PATH_CONTEXT**: Source path from where s2i command need to be run
+  (_default: ._).
+* **TLSVERIFY**: Verify the TLS on the registry endpoint (for push/pull to a
+  non-TLS registry) (_default:_ `true`)
+
+
+### Resources
+
+* **source**: A `git`-type `PipelineResource` specifying the location of the
+  source to build.
+
+## Outputs
+
+### Resources
+
+* **image**: An `image`-type `PipelineResource` specify the image that should
+  be built.
+
+## ServiceAccount
+
+S2I builds an image and pushes it to the destination registry which is
+defined as a parameter. In order to properly authenticate to the
+remote container registry, it needs to have the proper
+credentials. The credentials can be provided through a service
```

Comment:
Suggest: The image needs proper credentials to be authenticated by the remote container registry. These credentials can be...